### ISSUE TITLE
Add to package managers like scoop, winget and chocolatey

### ISSUE THREAD

**Issue by Gaweringo** [fougue/mayo#156]
I think it would be nice if this tool would be made available through some Windows package managers like [scoop](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets), [winget](https://learn.microsoft.com/en-gb/windows/package-manager/package/repository) and [chocolatey](https://docs.chocolatey.org/en-us/#create-packages).

Adding mayo to these package managers would make it (imo) a lot easier to install and update.

**Comment by HuguesDelorme** [comment]
@Silther
Yes seems to be added with PR https://github.com/microsoft/winget-pkgs/pull/95480

**Comment by HuguesDelorme** [comment]
Mayo 0.6.0 and 0.7.0(latest at the time of writing) versions are now available in winget-pkgs, see merged pull requests:
https://github.com/microsoft/winget-pkgs/pull/95480
https://github.com/microsoft/winget-pkgs/pull/119418

See also https://winstall.app/apps/Fougue.Mayo

**Comment by Mannshoch** [comment]
I detected that winget do find Mayo 0.8.0 and after install it get the problem to determine the installed mayo version.

```
Name        ID          Version Verfügbar Quelle
------------------------------------------------
Mayo v0.9.0 Fougue.Mayo Unknown 0.8.0     winget
```

Cause of that I missed the new version 0.9.0
[Edit] it appears the same after install v0.9.0

**Comment by HuguesDelorme** [comment]
> I detected that winget do find Mayo 0.8.0 and after install it get the problem to determine the installed mayo version.
> 
> ```
> Name        ID          Version Verfügbar Quelle
> ------------------------------------------------
> Mayo v0.9.0 Fougue.Mayo Unknown 0.8.0     winget
> ```
> 
> Cause of that I missed the new version 0.9.0 [Edit] it appears the same after install v0.9.0

@Mannshoch  That's weird because https://winstall.app/apps/Fougue.Mayo clearly states that v0.9.0 is latest version

**Comment by Mannshoch** [comment]
Yes Meanwhile the output changed:
```
PS C:\Windows\system32> winget update --include-unknown
Name                      ID                      Version      Available    Source
----------------------------------------------------------------------------------
Mayo v0.9.0               Fougue.Mayo             Unknown      0.9.0        winget

```

**Comment by dx-bhesse** [comment]
I think the issue is that the Mayo setup seems not to set the installed version in the Windows Registry.
In HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{F1978C7C-3C90-477F-B634-B99746AA153D}_is1 
"DisplayName" and "Publisher" exists, but "DisplayVersion" (should be set to "0.9.0") seems to be missing there and thats what winget probably reads to determine the installed version.

**Comment by HuguesDelorme** [comment]
> I think the issue is that the Mayo setup seems not to set the installed version in the Windows Registry. In HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall{F1978C7C-3C90-477F-B634-B99746AA153D}_is1 "DisplayName" and "Publisher" exists, but "DisplayVersion" (should be set to "0.9.0") seems to be missing there and thats what winget probably reads to determine the installed version.

Thanks for the info @dx-bhesse 
Will add and set the field `AppVersion=0.10.0` in the `[Setup]` section of the InnoSetup script file. Hopefully this will fix the issue for next version(which is coming soon)

**Comment by dx-bhesse** [comment]
Seems to have worked: "winget list" now shows "0.10.0" as version instead of "Unknown"
